resources/META-INF/plugin.xml

```diff
@@ -0,0 +1,55 @@
+<idea-plugin version="2">
+    <id>Batch Scripts Support</id>
+    <name>Batch Scripts Support</name>
+    <version>1.0.5</version>
+    <vendor>Alexey Efimov</vendor>
```

Comment:
No no, too much honour for a simple quick & dirty fix. :-) I do not even understand what the plugin does exactly and have never developed a plugin. This one is your baby, just mention me in the release noted, that's enough. I am just happy I managed to fix the plugin even though I use a deprecated method.